examples/cryptocurrency-advanced/backend/src/api.rs

```diff
@@ -50,28 +50,33 @@ pub struct WalletProof {
 /// Wallet history.
 #[derive(Debug, Serialize, Deserialize)]
 pub struct WalletHistory {
+    /// Proof to the transaction list.
     pub proof: ListProof<Hash>,
+    /// List of above transactions.
     pub transactions: Vec<WalletTransactions>,
 }
 
 /// Wallet information.
 #[derive(Debug, Serialize, Deserialize)]
 pub struct WalletInfo {
+    /// `BlockProof` of the last block.
```

Comment:
I don't like those doc comments as they repeat fields naming and their types. I want to see something more understandable by our end users